src/test/java/com/google/devtools/build/lib/skylark/StarlarkOptionsParsingTest.java

```diff
@@ -314,4 +319,16 @@ public void testOptionsAreParsedWithBuildTestsOnly() throws Exception {
 
     assertThat(result.getStarlarkOptions().get("//test:my_int_setting")).isEqualTo(15);
   }
+
+  @Test
+  public void testRemoveStarlarkOptionsWorks() throws Exception {
```

Comment:
Could you also add a test for the `--@//external/starlark/option` main repo case please?

Reply:
Yep! added